.github/workflows/buildpack-deps.yml

```diff
@@ -37,6 +37,8 @@ jobs:
 
       - name: comment PR
         if: "env.DOCKER_IMAGE"
-        uses: unsplash/comment-on-pr@b5610c6125a7197eaec80072ea35ef53e1fc6035 #v1.3.1
+        uses: unsplash/comment-on-pr@ffe8f97ccc63ce12c3c23c6885b169db67958d3b #v1.3.0
         with:
-          msg: "`${{ env.DOCKER_IMAGE }} ${{ env.DOCKER_REPO_DIGEST }}`."
\ No newline at end of file
+          msg: "`${{ env.DOCKER_IMAGE }} ${{ env.DOCKER_REPO_DIGEST }}`."
+          check_for_duplicate_msg: false
```

Comment:
Why don't we want this check? Won't the new hash be different anyway?

Reply:
I saw that yesterday the bot was only creating one comment (see https://github.com/ethereum/solidity/pull/13396), where it should have been created for. So I suspected that there is probably something wrong with this `check_for_duplicate_msg` - after changing this all comments showed up. But I did not do any further investigations on this.

Reply:
OK. I guess the worst that can happen is that we'll get some duplicate messages and we'll look into it if it happens.